mdtraj/geometry/rdf.py

```diff
@@ -91,3 +91,88 @@ def compute_rdf(traj, pairs, r_range=None, bin_width=0.005, n_bins=None,
+    For example, g(r, 0) is equal to the time-independent radial distribution function, g(r).
+    Please see https://doi.org/10.1103/PhysRev.95.249 for further reference.
+
+    Parameters
+    ----------
+    traj : Trajectory
+        Trajectory to compute time-dependent radial distribution function on.
+    pairs : array-like, shape=(n_pairs, 2), dtype=int
+        Each row gives the indices of two atoms.
+    times : array-like, shape=(any, 2), dtype=int
+        Each row gives the indices of two frames.
+    period_length : int, optional, default=None
+        The length of each chunk of frames to consider when time-averaging
+    r_range : array-like, shape=(2,), optional, default=(0.0, 1.0)
+        Minimum and maximum radii.
+    bin_width : float, optional, default=0.005
+        Width of the bins in nanometers.
+    n_bins : int, optional, default=None
+        The number of bins. If specified, this will override the `bin_width`
+         parameter.
+    self_correlation : bool, default=True
+        Whether or not to include the self-correlation, the case of i=j
+    periodic : bool, default=True
+        If `periodic` is True and the trajectory contains unitcell
+        information, we will compute distances under the minimum image
+        convention.
+    opt : bool, default=True
+        Use an optimized native library to compute the pair wise distances.
+
+    Returns
+    -------
+    r : np.ndarray, shape=(np.diff(r_range) / bin_width - 1), dtype=float
+        Radii values corresponding to the centers of the bins.
+    g_r_t : np.ndarray, shape=(len(times), np.diff(r_range) / bin_width - 1), dtype=float
+        Radial distribution function values at r for each time pair.
+
+    See also
+    --------
+    Topology.select_pairs
+
+    """
+    if r_range is None:
+        r_range = np.array([0.0, 1.0])
+    r_range = ensure_type(r_range, dtype=np.float64, ndim=1, name='r_range',
+                          shape=(2,), warn_on_cast=False)
+    if n_bins is not None:
+        n_bins = int(n_bins)
+        if n_bins <= 0:
+            raise ValueError('`n_bins` must be a positive integer')
+    else:
+        n_bins = int((r_range[1] - r_range[0]) / bin_width)
+
+    if period_length is None:
+        period_length = traj.n_frames
+
+    # Add self pairs to `pairs`
+    if self_correlation:
+        pairs_set = np.unique(pairs)
+        pairs = np.vstack([np.vstack([pairs_set, pairs_set]).T, pairs])
+
+    g_r_t = np.zeros(shape=(len(times), n_bins))
+    num_chunks = int(np.floor(traj.n_frames / period_length))
+
+    # Returns shape (len(times), len(pairs))
+    frame_distances = compute_distances_t(traj, pairs, times, periodic=periodic, opt=opt)
+
+    for n, distances in enumerate(frame_distances):
+        tmp, edges = np.histogram(distances, range=r_range, bins=n_bins)
+        g_r_t[n, :] += tmp
+    r = 0.5 * (edges[1:] + edges[:-1])
+
+    # Normalize by volume of the spherical shell (see above)
```

Comment:
I don't know what "see above" refers to - the similar lines in `compute_rdf`? There's a decent chance I wrote this comment, so if I can't remember that should be added

Reply:
I believe this refers to lines 84-89 which point to a discussion regarding the normalization through the volume of a spherical shell.